views/__init__.py

```diff
@@ -180,6 +180,11 @@ def login():
         return jsonify(success="Authenticated"), 200
 
 # 
+@app.route('/login', methods=['GET'])
+def login_form():
+    return redirect('/#login')
```

Comment:
tbh not sure why that if statement is there but I suspect that it's there because redirecting to the home page caused some issues on the server.